api/src/opentrons/protocol_api/core/protocol_api/module_geometry.py

```diff
@@ -170,7 +169,7 @@ def highest_z(self) -> float:
             return self._height
 
     def __repr__(self) -> str:
-        return self._display_name
+        return f"{self._display_name}{' on '+str(self.parent) if self.parent else ''}"
```

Comment:
Lets try to avoid f-string weirdness like this. Use variables to avoid the quirky stuff that Python [hasn't fixed yet](https://peps.python.org/pep-0701/) 

```suggestion
        location = f" on {self.parent}" if isinstance(self.parent, str) else ""
        return f"{self._display_name}{location}"
```